monai/apps/auto3d/hpowrapper/nni/nni_wrapper.py

```diff
@@ -0,0 +1,35 @@
+import nni
+import sys
+sys.path.append('..')
+from dummy_trainner import trainner
+from hpo_wrapper import HPO_wrapper
+
+class NNI_wrapper(HPO_wrapper):
+    def __init__(self, trainner=None, **kwargs):
+        super().__init__(trainner, **kwargs)
+
+    def _get_hyperparameters(self):
+        return nni.get_next_parameter()
+
+    def _update_model(self, params):
+        self.trainner.update(params)
+
+    def run(self):
```

Comment:
As discussed with @mingxin-zheng , we should abstract the `execution` progress of Auto3DSeg.
And the NNI run is another run method, similar to the typical run from history.
https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_auto3dseg_ensemble.py#L117-L120

Thanks.